packages/client/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@webcomponents/internal-site-client",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Client JavaScript used on webcomponents.org",
+  "author": "Google LLC",
+  "license": "Apache-2.0",
+  "type": "module",
+  "scripts": {
+    "build": "wireit",
+    "build:dev": "wireit",
+    "build:prod": "wireit",
+    "check:types": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:dev",
+        "build:prod",
+        "check:types"
+      ]
+    },
+    "build:dev": {
+      "command": "esbuild --format=esm --target=es2022 --color --outdir=lib src/**/*.ts ",
```

Comment:
Are we using esbuild here just because it's faster than `tsc`?

How quickly does it keep up with TypeScript? When could we start using 4.9 and the `satisfies` operator?

Reply:
Yeah, esbuild is just SO much faster than tsc. It's so much better for the dev cycle, because you save your ts file, and the browser instantly updates. (Right now we don't have a lot of code, so the difference is small, but over time the difference will become large).

I don't know how quickly it keeps up with TypeScript. I would guess pretty good, but yeah that's something to keep in mind. An option we do have down the line is to use esbuild for dev code, and TypeScript for the prod build, e.g. if we preferred the output for some reason (but that doesn't address unsupported syntax).